src/vmm/src/vstate/vm.rs

```diff
@@ -304,14 +304,14 @@ pub(crate) mod tests {
     use crate::vstate::system::KvmContext;
     use vm_memory::GuestAddress;
 
-    #[cfg(target_arch = "x86_64")]
     // Auxiliary function being used throughout the tests.
-    fn setup_vm(mem_size: usize) -> (Vm, GuestMemoryMmap) {
+    pub(crate) fn setup_vm(mem_size: usize) -> (Vm, GuestMemoryMmap) {
         let kvm = KvmContext::new().unwrap();
         let gm = GuestMemoryMmap::from_ranges(&[(GuestAddress(0), mem_size)]).unwrap();
+        //let err = Vm::new(kvm.fd());
+        //println!("err is {}", err.err().unwrap());
```

Comment:
Nit: is this comment needed ?

Reply:
No, no, I'll delete it when I do the rebase and get a second review.

Reply:
Done, take a look.